assets/js/modules/pagespeed-insights/components/ReportDetailsLink.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Report Details Link component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __, _x } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { sanitizeHTML } from '../../../util';
+const { useSelect } = Data;
+
+export default function ReportDetailsLink() {
+	const referenceURL = useSelect( ( select ) => select( CORE_SITE ).getCurrentReferenceURL() );
+	const pagespeedInsightsURL = addQueryArgs( 'https://developers.google.com/speed/pagespeed/insights/', { url: referenceURL } );
+	const footerLinkHTML = sprintf(
+		/* translators: 1: link attributes, 2: translated service name */
+		__( 'View details at <a %1$s>%2$s</a>', 'google-site-kit' ),
+		`href="${ pagespeedInsightsURL }" class="googlesitekit-cta-link googlesitekit-cta-link--external" target="_blank"`,
```

Comment:
Do we really need to be using `target="_blank"` here? It's a lousy experience on any device, especially mobile, it presents accessibility problems, and is really only defensible if the navigation action will lead to data loss (eg the user is completing a form we don't save and then navigate away with a link). Savvy users can _choose_ to open links in a new tab, but can't choose _not_ to when we set this attribute.

I don't think the link in this context does anything like that, right? If the user can click "back" after navigating to that link we should ditch the `target="_blank"` here.

Reply:
This shouldn't be changed at this point. While I'm aware of the accessibility implications of this, we have these as external links elsewhere. At a minimum, even if we removed the `target="_blank"`, this should also have the external link indicator removed.